Makefile

```diff
@@ -262,10 +262,19 @@ amazon-cluster:
 	aws s3api create-bucket --bucket $(BUCKET_NAME) --region $(AWS_REGION)
 	aws ec2 create-volume --size $(STORAGE_SIZE) --region $(AWS_REGION) --availability-zone $(AWS_AVAILABILITY_ZONE) --volume-type gp2
 
-clean-amazon-cluster:
+amazon-clean-cluster:
```

Comment:
Since this command permanently deletes data, can we put a simple command-line prompt here?  Something like "The following script will delete your AWS bucket and volume.  The action cannot be undone.  Do you want to proceed?  (Y/n)"